### ISSUE TITLE
Any difficulties when apply it some other GNN models?

### ISSUE THREAD

**Issue by cshjin** [abojchevski/graph_cert#3]
Hi @abojchevski,
I see the model is based on personalized PageRank. 
However, is it also applicable to other GNN models? Based on minimizing the logits of the output layer.

Any problems, for example, solving the linear system or the nonlinearity inside the model itself?

**Comment by abojchevski** [comment]
The certificate can be applied only to models where the predictions are a linear function of (personalized) PageRank. This include PiPPNP and label/feature propagation. There might be some way to extend this to other GNN models, e.g. by bounding the output using PageRank but this is not straightforward as far as I can tell.

**Comment by cshjin** [comment]
Yes, that's causing the same concern to me. Thanks.
